lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -472,9 +472,7 @@ public function GetQueueContents(array $params)
 
         if ($result = $this->db->GetAll(
             'SELECT DISTINCT t.id, t.customerid, t.address_id, va.name AS vaname, va.city AS vacity, va.street, va.house, va.flat, c.address, c.city, vusers.name AS ownername,
-				t.subject, t.state, owner AS ownerid, t.requestor AS req, t.source, t.priority, rtqueues.name, t.requestor_phone, t.requestor_mail, t.deadline, t.requestor_userid,
-				CASE WHEN customerid IS NULL THEN t.requestor ELSE '
-                . $this->db->Concat('c.lastname', "' '", 'c.name') . ' END AS requestor,
+				t.subject, t.state, owner AS ownerid, t.requestor AS req, t.source, t.priority, rtqueues.name, t.requestor_phone, t.requestor_mail, t.deadline, t.requestor_userid, c.name, c.lastname, rq.name AS requestor_name,
```

Comment:
Dlaczego zwijasz to do jednego, dłuższego wiersza?

Reply:
odwinąłem mu